vmngclient/api/software_action_api.py

```diff
@@ -182,7 +191,7 @@ def upgrade_software(
             ],  # type: ignore
             "deviceType": install_spec.device_type.value,
         }
-        if install_spec.family.value in (DeviceClass.VMANAGE.value, DeviceClass.CEDGE.value):
+        if personality in (Personality.VMANAGE, Personality.EDGE):  # block also vedges
```

Comment:
What is the purpose of this comment?
Maybe no comment or something like `block downgrade for edges and vmanages`?